src/main/js/main_html5.js

```diff
@@ -336,8 +336,13 @@ require("../../../html5-common/js/utils/environment.js");
           _video.pause();
         } catch (ex) {
           // error because currentTime does not exist because stream hasn't been retrieved yet
-          console.log('VTC_OO: Failed to rewind video, probably ok; continuing');
+          OO.log('VTC_OO: Failed to rewind video, probably ok; continuing');
         }
+      } else if (!!rewind && OO.isEdge) {
+        // PBW-4555: Edge browser will always go back to time 0 on load.  Setting time to 0 here would
+        // cause the raw video element to enter seeking state.  If we call load while seeking on Edge,
+        // then seeking no longer works until the video stream url is changed.
+        currentTime = 0;
```

Comment:
I would recommend to consolidate code under if (!!rewind) clause:
{code}
if (!!rewind) {
  if (!OO.isEdge) {  // consider adding loaded &&
    try {
      if (OO.isIos && OO.iosMajorVersion == 8) {
        // On iOS, wait for durationChange before setting currenttime
        $(_video).on("durationchange", _.bind(function() {
          _video.currentTime = 0;
          currentTime = 0;
        }, this));
      } else {
        _video.currentTime = 0;
        currentTime = 0;
      }
      _video.pause();
    } catch (ex) {
      // error because currentTime does not exist because stream hasn't been retrieved yet
      OO.log('VTC_OO: Failed to rewind video, probably ok; continuing');
    }
  } else {
    // PBW-4555: Edge browser will always go back to time 0 on load.  Setting time to 0 here would
    // cause the raw video element to enter seeking state.  If we call load while seeking on Edge,
    // then seeking no longer works until the video stream url is changed.
    currentTime = 0;
  }
}
{code}